### ISSUE TITLE
Adding screenshots for Android & Flutter projects 

### ISSUE THREAD

**Issue by SidharthMudgil** [SidharthMudgil/mini-projects#1]
Show the screenshot & demo for each Android and Flutter projects.

**Comment by shivam0277** [comment]
i would like to work on this issue pls assign it to me.

**Comment by SidharthMudgil** [comment]
@shivam0277  I've assigned the issue... check the compatitive programming repo of mine for checking how I wanted navigation and for mini projects Android, flutter... just add title 2-3 line about app and screenshots in the table like added in mosam repository of mine.

**Comment by SidharthMudgil** [comment]
@shivam0277 Any update or I'm removing your assignment.

**Comment by JTateCC** [comment]
I am available to work on this under the hacktober fest tag

**Comment by SidharthMudgil** [comment]
@JTateCC  so you will be working from 1st october, Right??

**Comment by JTateCC** [comment]
> @JTateCC so you will be working from 1st october, Right??

Will work from now but make the PR on october 1st

**Comment by Riyagupta0204** [comment]
can you assign  this to me?

**Comment by SidharthMudgil** [comment]
Sorry this issue is already assigned. Otherwise it'll cause conflicts. I'll you let you know if there is any other documentation related task

**Comment by SidharthMudgil** [comment]
@JTateCC are you working on the issue? any progress

**Comment by JTateCC** [comment]
> @JTateCC are you working on the issue? any progress

Yep have got about half the screenshots I want. I will share something end of day

**Comment by JTateCC** [comment]
I am hvng issues with android studio so please free to unassign me and assign to someone else.

**Comment by SidharthMudgil** [comment]
@Riyagupta0204 do you want to work on it?

**Comment by recruitchaos** [comment]
Would like to work on this issue, can you assign this to me ?

**Comment by SidharthMudgil** [comment]
Assigned

**Comment by recruitchaos** [comment]
can you add hacktober tag for this

**Comment by recruitchaos** [comment]
do I have to add screenshots only for the flutter projects or like we are doing for other projects?

**Comment by SidharthMudgil** [comment]
Flutter and Android.... add like title then small description then screenshots in table form

**Comment by recruitchaos** [comment]
<img width="1680" alt="Screenshot 2023-10-06 at 1 52 11 PM" src="https://github.com/SidharthMudgil/mini-projects/assets/93324285/4eebc641-e31c-4b43-afc9-d6cab448eb54">

Looks good ?

**Comment by SidharthMudgil** [comment]
Awesome 🙌

**Comment by SidharthMudgil** [comment]
Just change title  like "Expense Tracker"....

**Comment by recruitchaos** [comment]
Alright!

**Comment by recruitchaos** [comment]
there is not much content for some flutter apps

**Comment by SidharthMudgil** [comment]
No worry add single screenshot if there is single screen...

**Comment by SidharthMudgil** [comment]
still working?

**Comment by recruitchaos** [comment]
yes! taking this much time due to my semester exams 🙂

**Comment by recruitchaos** [comment]
hi, i am encountering problem while running other than flutter projects; flutter projects readme are done !
can I take more time?

**Comment by SidharthMudgil** [comment]
Yeah, take your time..

**Comment by recruitchaos** [comment]
thanks!
